commission/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 # Copyright 2014-2022 Tecnativa - Pedro M. Baeza
 {
     "name": "Commissions",
-    "version": "17.0.1.1.0",
+    "version": "18.0.1.1.0",
```

Comment:
new version should start as 18.0.1.0.0

Reply:
Hello, this change is in the last commit of this PR

Reply:
The commit url is the following

https://github.com/OCA/commission/commit/756b3528f40fb14c834109c294ffa3bf3d4df309

I detected an error, it must be 18.0.1.0.0 and is defined as 18.0.1.1.0 do I will update the PR

Reply:
Now I get it. Sorry, I will update it

Reply:
@alan196 are you planning to update this one?